karapace/protobuf/field_element.py

```diff
@@ -142,21 +151,26 @@ def compare_type(
     def compare_message(
         cls, self_type: ProtoType, other_type: ProtoType, result: CompareResult, types: CompareTypes
     ) -> None:
-        from karapace.protobuf.message_element import MessageElement
+        from karapace.protobuf.type_element import TypeElement
```

Comment:
Why imported at this point?

Reply:
Not sure, fixed now. Maybe there was some thing about circular dependencies...